src/main/java/com/amazon/opendistroforelasticsearch/security/resolver/IndexResolverReplacer.java

```diff
@@ -316,26 +287,61 @@ public boolean replace(final TransportRequest request, boolean retainMode, Strin
         }, false);
     }
 
+    static final class IndexResolveKey {
```

Comment:
nit: please declare the class and all member variables `private`.

Reply:
Done